File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();
    public PageResult findBrandByPage(int pageNum, int pageSize);
    public int add(TbBrand tbBrand);

    public TbBrand findOne(long id);
    public int update(TbBrand tbBrand);
    public void delete(long[] ids);
    public PageResult search(TbBrand tbBrand,int pageNum, int pageSize);
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.search=function (page,size,searchEntity) {
        return $http.post('../brand/search.do?page='+page +'&size='+size, searchEntity)
    };

    this.add=function (entity) {
        return $http.post('../brand/add.do',entity)
    };
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity)
    };
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }
    this.delete=function (ids) {
        return $http.post('../brand/delete.do?ids='+ids)
    }

});